rruntime/goroutine-factory.go

```diff
@@ -23,16 +19,14 @@ import (
 //	})
 func Go(function func()) {
 	go func() {
-		ctx := bugsnag.StartSession(context.Background())
-		defer misc.BugsnagNotify(ctx, "Core")()
+		defer crash.Notify("Core")()
 		function()
 	}()
 }
 
 func GoForWarehouse(function func()) {
 	go func() {
-		ctx := bugsnag.StartSession(context.Background())
-		defer misc.BugsnagNotify(ctx, "Warehouse")()
+		defer crash.Notify("Warehouse")()
```

Comment:
Can't we use the wrapper here?

Reply:
Same for core.

Reply:
The wrapper requires a diffrent signature for function, func() error.